openedx/features/learner_profile/static/learner_profile/templates/learner_profile.underscore

```diff
@@ -1,15 +1,25 @@
 <div class="profile <%- ownProfile ? 'profile-self' : 'profile-other' %>">
     <div class="wrapper-profile-field-account-privacy"></div>
     <div class="wrapper-profile-sections account-settings-container">
-        <div class="wrapper-profile-section-one">
-            <div class="profile-image-field">
+        <% if (ownProfile) { %>
+            <div class="profile-header">
+                <div class="header"> <%- profile_header %></div>
+                <div class="subheader"> <%- profile_subheader %></div>
             </div>
-            <div class="profile-section-one-fields">
+        <% } %>
+        <div class="wrapper-profile-section-container-one">
```

Comment:
This is a nit.  You don't have to do this, but you should respond if you plan to skip.